communication/include/communication/CommStateControl.hpp

```diff
@@ -11,6 +11,8 @@
 // LICENSE file.
 
 #pragma once
+#include <functional>
+#include "callback_registry.hpp"
 namespace bft::communication {
 class CommStateControl {
```

Comment:
it's in `bft::communication` namespace (why not `concord::bft::communication`?), so `Comm` is redundant

Reply:
fixed